dashboard/src/utils/filters.ts

```diff
@@ -45,3 +51,25 @@ export const getTargetFilter = (
 
   return acc;
 };
+
+export const useDiffFilterParams = (
```

Comment:
this will work only in the TreeDetailsPage
so I woudl move it to  `pages/TreeDetails/utils/treeDetailsFilters.ts`